packages/template-retail-react-app/app/theme/components/base/button.js

```diff
@@ -42,20 +42,24 @@ export default {
             color: 'black',
             justifyContent: 'flex-start',
             fontSize: 'sm',
-            _hover: {bg: 'gray.50', textDecoration: 'none'},
+            _hover: {bg: 'gray.50', textDecoration: 'underline', textDecorationColor: '#181818'},
```

Comment:
```suggestion
            _hover: {bg: 'gray.50', textDecoration: 'underline', textDecorationColor: 'gray.900'},
```

Reply:
Suggestion applied. I also replaced the blues with `blue.600`